### PR TITLE
feat: add jsonReadOnly prop

### PR DESCRIPTION
**Pull request by fabien-roy**
`mjmlReadOnly` prop was added and it's really useful. In our case, having `jsonReadOnly` would also be a good addition, since we want to control that.

- Add `jsonReadOnly` prop, similar to `mjmlReadOnly`
- Add readonly styles for text areas (remove focus/hover states, use disabled color text)

### CODE REVIEW

**Comment by m-Ryan** [comment]
@fabien-roy  Thank you for your PR, please check the comments of the review.

**Review by m-Ryan — APPROVED** [review]

**Review by m-Ryan — CHANGES_REQUESTED** [review]
Please check

File: packages/easy-email-extensions/src/index.scss [hunk 1]
```diff
@@ -26,4 +26,16 @@
...
+    color: var(--color-text-2);
+    &:hover {
+      border-color: var(--color-neutral-3);
+    }
+    &:focus {
+      border-color: var(--color-neutral-3);
+      background-color: transparent;
+      box-shadow: none;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (fabien-roy) [reply packages/easy-email-extensions/src/index.scss:40]: My mistake, thanks!

File: packages/easy-email-extensions/src/StandardLayout/StandardLayout.tsx [hunk 2]
```diff
@@ -92,6 +92,7 @@ export const StandardLayout: React.FC<ExtensionProps> = props => {
     showSourceCode = true,
     compact = true,
     categories = defaultCategories,
+    jsonReadOnly = true,
```
[the remark this answers is not in the document]
Comment (fabien-roy) [reply packages/easy-email-extensions/src/StandardLayout/StandardLayout.tsx:95]: Right, disabled by default sounds like a better idea. I'll do that!

**Review by m-Ryan — APPROVED** [review]
LGBM

**Comment by m-Ryan** [comment]
Released at  - easy-email-extensions@4.12.3

**Pull request merged**